src/components/DateInput/dateInput.stories.mdx

```diff
@@ -42,14 +42,14 @@ prop.
 ## Default selected date
 
 The input element in `DateInput` will be empty by default.
-Pass an instance of `Date` or a parsable date string (`MM/DD/YYYY`) to the `defaultDate` prop to set a default value.
+Due to inconsistency across browsers with how native `Date` parsing works, we recommend passing a string to `defaultDate`.
```

Comment:
Would we consider getting rid of this sentence completely?

Reply:
I'd like to keep it in, as the prop types table still declares `Date` as valid for this prop

Reply:
I understand the information it conveys but from the consumer perspective they don't actually know this accepts a native date object unless we tell them.

Reply:
We unfortunately do tell them, in the props table, until we remove support in a major version

Reply:
makes sense 👍